src/Pages/Detail/Aside.js

```diff
@@ -0,0 +1,289 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import styled, { css } from "styled-components";
+import { useHistory } from "react-router-dom";
+import {
+  AiOutlineHeart,
+  AiOutlineYoutube,
+  AiFillGift,
+  AiOutlineUser,
+  AiOutlineLike,
+  AiFillHeart,
+} from "react-icons/ai";
+import Button from "./Components/Button";
+import { Hr, API, Tooltip } from "./Components/Utils";
+
+const Aside = () => {
+  const [category, setCategory] = useState("");
+  const [creatorName, setCreatorName] = useState("");
+  const [classTitle, setClassTitle] = useState("");
+  const [price, setPrice] = useState("");
+  const [heart, setHeart] = useState(0);
+  const [heartToggle, setHeartToggle] = useState(false);
+  const [level, setLevel] = useState("");
+  const history = useHistory();
+
+  useEffect(() => {
+    axios.get(API).then((res) => setLevel(res.data.detail.class_info.level));
+  }, []);
+
+  const contentItem = [
+    [<AiOutlineYoutube size="18" />, <span>콘텐츠 이용권</span>],
+    [<AiFillGift size="18" />, <span>준비물 키트</span>],
+    [<AiOutlineUser size="18" />, <span>{level} 대상</span>],
+    [<AiOutlineLike size="18" />, <span>개인 만족도</span>],
+  ];
```

Comment:
```suggestion
  const contentItems = [
    [<AiOutlineYoutube size="18" />, <span>콘텐츠 이용권</span>],
    [<AiFillGift size="18" />, <span>준비물 키트</span>],
    [<AiOutlineUser size="18" />, <span>{level} 대상</span>],
    [<AiOutlineLike size="18" />, <span>개인 만족도</span>],
  ];
```
- `s`를 붙여 복수형 자료임을 표기해주세요
- 함수 컴포넌트 바깥 쪽으로 분리해도 되는 순수한 내용인 것 같습니다. 매 렌더 때마다 할당될 필요 없으므로 분리해주세요.

Reply:
level부분이 state값에 의해 바뀌어서 컴포넌트 안에 있어야하지 않나요?